### PR TITLE
Add postprocess option

### PR DESCRIPTION
**Pull request by arturopuente**

### CODE REVIEW

**Comment by blackjid** [comment]
Nice! I like this!!

I have two thoughts about it..

- the `add_recipe` command: I don't like to have to use an underscore in the cli. It's weird. I rather prefer to have something that follow the git commands convention. Something like `potassium recipe add i18n`. 

- the **recipe** name: I don't like how the term recipe is being used. :grin: I think you use a recipe to build something else. So it's perfect to be used internally but not as the command exposed through the cli. I'd prefer something like `potassium feature add i18n` for example

**Comment by blackjid** [comment]
After write this, I was thinking that maybe (just maybe) an alternative that could work better would be to have an install command. `potassium install i18n` o `potassium install api`

**Comment by emilioeduardob** [comment]
@blackjid listo renombrado a `install` por arturo. Le puse que si no especificas una receta, te muestre la lista con inquirer y puedas elegir y ejecutarla

**Comment by emilioeduardob** [comment]
:eyes: @bunzli

**Comment by bunzli** [comment]
estoy de acuerdo con el cambio de nombre que propuso el negro y según puedo ver funciona bien. Es un honor que se me haya asignado este PR, pero creo que @iobaixas podría aportar una mirada más crítica sobre la implementación.

File: lib/potassium/templates/application/helpers/recipe-helpers.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module RecipeHelpers
+  def export(file)
+    exported_recipes << File.basename(file, '.rb')
```
Comment (juliogarciag) [inline lib/potassium/templates/application/helpers/recipe-helpers.rb]: No veo que se uste helper o el objeto `exported_recipes`.

**Pull request merged**